### PR TITLE
Add cbrt function to the PPL

### PR DESCRIPTION
**Pull request by margarit-h**
* Add cbrt function to the PPL

Signed-off-by: Margarit Hakobyan <margarith@bitquilltech.com>

### Description
Calculates the cube root of a number

Argument type: INTEGER/LONG/FLOAT/DOUBLE

Return type DOUBLE:

(Non-negative) INTEGER/LONG/FLOAT/DOUBLE -> DOUBLE
(Negative) INTEGER/LONG/FLOAT/DOUBLE -> DOUBLE

Example::

    opensearchsql> source=location | eval `CBRT(8)` = CBRT(8), `CBRT(9.261)` = CBRT(9.261), `CBRT(-27)` = CBRT(-27) | fields `CBRT(8)`, `CBRT(9.261)`,`CBRT(-27)`;
    fetched rows / total rows = 2/2
    +-----------+---------------+-------------+
    | CBRT(8)   | CBRT(9.261)   | CBRT(-27)   |
    |-----------+---------------+-------------|
    | 2.0       | 2.1           | -3.0        |
    | 2.0       | 2.1           | -3.0        |
    +-----------+---------------+-------------+
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [x] New functionality has been documented.
  - [x] New functionality has javadoc added
  - [x] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: integ-test/src/test/java/org/opensearch/sql/ppl/MathematicalFunctionIT.java [hunk 1]
```diff
@@ -262,6 +262,19 @@ public void testSqrt() throws IOException {
...
+  public void testCbrt() throws IOException {
+    JSONObject result =
+            executeQuery(
+                    String.format(
+                            "source=%s | eval f = cbrt(age) | fields f", TEST_INDEX_BANK));
+    verifySchema(result, schema("f", null, "double"));
+    verifyDataRows(result,
+            rows(3.174802103936399), rows(3.3019272488946267), rows(3.0365889718756627),
+            rows(3.2075343299958265), rows(3.3019272488946267), rows(3.391211443014167),
+            rows(3.2396118012774835));
```
Comment (dai-chen) [inline integ-test/src/test/java/org/opensearch/sql/ppl/MathematicalFunctionIT.java:275]: I want to call out the same concern in previous PR: please double check if floating point precision problem may happen. I don't see `rows()` handle this similar as `MatcherUtils.closeTo()` matcher.

Comment (margarit-h) [reply integ-test/src/test/java/org/opensearch/sql/ppl/MathematicalFunctionIT.java:277]: Feedback addressed in b60195c90a24932f36c65c9c12c869b63ab8b5de

Comment (dai-chen) [reply integ-test/src/test/java/org/opensearch/sql/ppl/MathematicalFunctionIT.java:277]: Thanks for asserting by closeTo below. Should we remove test cases here?

Comment (margarit-h) [reply integ-test/src/test/java/org/opensearch/sql/ppl/MathematicalFunctionIT.java]: Removed in 77de5e53e04e701e14124222614826169bf57191 commit

**Review by dai-chen — APPROVED** [review]
Thanks for the changes!

**Pull request merged**